### PR TITLE
Add Restarts section in Managing Apps section

### PR DESCRIPTION
**Pull request by jonathanglassman**
## What

- Created restarts markdown file with content on CF restarting apps outside of your direct control.
- Added new restarts file into index file.

## How to review

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).
1. Check this fulfils requirements of story: https://www.pivotaltracker.com/story/show/155904229

## Who can review

Anyone except Jon Glassman or Sam Crang

### CODE REVIEW

File: source/documentation/managing_apps/restarts.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Cloud Foundry will restart your app due to circumstances outside of your direct control, such as:
+
+- platform upgrades
+- failure of the underlying hardware platform
+- applying operating system patches to virtual machines that host app instances
+
+Cloud Foundry sends a termination signal (SIGTERM) to any apps it wants to restart. To shut down cleanly, apps must finish any requests within 10 seconds of receiving the termination signal. If the app does not respond in time, Cloud Foundry will force-terminate the app through a SIGKILL signal without shutting the app down cleanly, and requests may fail. 
+
+If your app has more than one instance, the running instances will not all restart at the same time. This ensures there is always at least one available instance for your app.
+
```
Comment (venusbb) [inline source/documentation/managing_apps/restarts.md]: I suspect that this restarting is done one by one. We need to give a clearer picture on how this work. If an app is having 50 instances running - going down to one, is a scary thought.

File: source/documentation/managing_apps/restarts.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+For more information, see the Cloud Foundry documentation on [evacuation](https://docs.cloudfoundry.org/devguide/deploy-apps/app-lifecycle.html#evacuation).
```
Comment (venusbb) [inline source/documentation/managing_apps/restarts.md]: I clicked on the link - it doesn't give any more information about what's happening. I wonder if there is a better reference that tells us more about it.

File: source/documentation/managing_apps/restarts.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## App restarts
+
+Cloud Foundry will restart your app due to circumstances outside of your direct control, such as:
+
+- platform upgrades
+- failure of the underlying hardware platform
+- applying operating system patches to virtual machines that host app instances
+
+Cloud Foundry sends a termination signal (SIGTERM) to any apps it wants to restart. To shut down cleanly, apps must finish any requests within 10 seconds of receiving the termination signal. If the app does not respond in time, Cloud Foundry will force-terminate the app through a SIGKILL signal without shutting the app down cleanly, and requests may fail.
```
Comment (venusbb) [inline source/documentation/managing_apps/restarts.md]: Can be more explicit what the tenant should do if they have processes that run for more than 10 seconds.

File: source/documentation/managing_apps/restarts.md (hunk 2 above)
Comment (venusbb) [reply source/documentation/managing_apps/restarts.md]: [diego upgrade](https://docs.cloudfoundry.org/adminguide/diego-cell-upgrade.html)

Comment (alext) [reply source/documentation/managing_apps/restarts.md]: That page is from the admin guide which is aimed at platform operators. I'm not sure that's the best place to be directing end-users of the platform...

What additional information are you wanting that's not included in the existing link?

File: source/documentation/managing_apps/restarts.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+## App restarts
+
+Cloud Foundry will restart your app due to circumstances outside of your direct control, such as:
+
+- platform upgrades
+- failure of the underlying hardware platform
+- applying operating system patches to virtual machines that host app instances
+
```
Comment (venusbb) [inline source/documentation/managing_apps/restarts.md]: - your app instance runs out of memory or disk space

**Pull request merged**